src/sentry/static/sentry/app/types/dynamicSampling.tsx

```diff
@@ -17,6 +17,10 @@ export enum DynamicSamplingConditionOperator {
   OR = 'or',
   AND = 'and',
   NOT = 'not',
+  /**
+   * It uses all other operators
+   */
+  ALL = 'all',
```

Comment:
this should not be part of this PR

Reply:
Yeah I don't know why these are showing up in the diffs even after merging with master.